### ISSUE TITLE
Visual Indicators (Affordances)

### ISSUE THREAD

**Issue by rachaelbradley** [w3c/wcag#1124]
The AG group acknowledged the importance of Visual Indicators (Affordances) but was unable to find an acceptable approach.  

The purpose of this success criterion is to help people with cognitive disabilities or low vision to easily identify controls. 

This issue captures the documentation so that AG can continue to work on this in the next iteration of WCAG. 
[Visual Indicators History.docx](https://github.com/w3c/wcag/files/4622640/Visual.Indicators.History.docx)
